wandb/tensorboard/watcher.py

```diff
@@ -5,6 +5,8 @@
 for path in sys.path:
     if "client/wandb" in path:
         sys.path.remove(path)
+if sys.modules.get("tensorboard"):
+    del sys.modules["tensorboard"]
```

Comment:
Whoa, nice find.

Reply:
It was sitting in my tensorboard watcher WIP branch:
https://github.com/wandb/client/pull/585
Will try and get that revived and ready for review in the next couple weeks.

Reply:
@vanpelt , can you take another look?  I had to do another workaround for `pip install ... #egg=wandb` python environments.

Reply:
Cool, looks good.